tests/test_cli.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+import subprocess
+import sys
+
+import pytest
+
+from trustme._cli import main
+
+
+def test_trustme_cli(tmpdir):
+    with tmpdir.as_cwd():
+        main(argv=[])
+
+    assert tmpdir.join("server.key").check(exists=1)
+    assert tmpdir.join("server.pem").check(exists=1)
+    assert tmpdir.join("client.pem").check(exists=1)
```

Comment:
Not mandatory since the cli is quite simple right now, but trustme has handy helpers that would allow us to make sure those files contain what we want.